scripts/s3-support-bundles/s3_bundle_generate.sh

```diff
@@ -56,7 +56,6 @@ s3server_binary="/opt/seagate/cortx/s3/bin/s3server"
 s3_motr_dir="/var/motr/s3server-*"
 s3_core_dir="/var/crash"
 sys_auditlog_dir="/var/log/audit"
-s3_recovery_dir="/var/log/seagate/s3/s3recovery"
```

Comment:
Recovery tool log directory is already collected as part of s3 directory (/var/log/seagate/s3) .. hence no need to collect it explicitly here.